test/lib/yggdrasil/command/parser_test.exs

```diff
@@ -0,0 +1,17 @@
+defmodule ParserTest do
```

Comment:
@matthewphilyaw things are starting to come together now. Did a bit of clean up and added these tests.